install.mdx

```diff
@@ -83,6 +74,25 @@ To continue please select in which environment you are running your Postgres dat
     EnterpriseDB
   </Link>*/}
```

Comment:
Theoretically we should be able to remove these commented-out tiles, but removing them breaks all images on the page. :confounded: